src/main/java/core/basesyntax/DrawingFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawingFigure {
```

Comment:
According to the checklist, interface names should not use verbs. Instead of `DrawingFigure`, a name like `FigureDrawer` would be more appropriate as it follows the convention of using nouns for class/interface names.

Reply:
rename FigureDrawer or even better, Drawable